src/models/ComponentAttribute.js

```diff
@@ -4,12 +4,12 @@
 class ComponentAttribute {
   /**
    * Default constructor.
-   * @param {object} [props={}] - Object that contains all properties to set.
-   * @param {string} [props.name=null] - The name of the attribute.
-   * @param {(string | boolean | number | Array | ComponentAttribute[])} [props.value=null] - The
+   * @param {object} [props] - Object that contains all properties to set.
+   * @param {string} [props.name] - The name of the attribute.
+   * @param {(string | boolean | number | Array | ComponentAttribute[])} [props.value] - The
```

Comment:
(string | boolean | number | array | ComponentAttribute[] | null)

Reply:
`null` is not a type, so no changes to be done here